maintenance/Terminal.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Voron Commands
+layout: default
+parent: Maintenance Resources
+nav_order: 2
+---
+
+# Voron Maintenance Commands
+
+This document provides a list of commonly used commands for maintaining and troubleshooting Voron 3D printers.
+
+## General Commands
+
+- **FIRMWARE_RESTART**  
+  Use this command if any updates are made to the printer configuration file.  
+  1. Save the configuration file.  
+  2. Restart Klipper by typing:  
+  `FIRMWARE_RESTART`
+
+- **Restart Klipper**  
+  If you need to restart Klipper, click on:  
+  `restart Klipper`
+   <p align="center">
+     <img src="attachments/restart.jpg" width="275" height="197" alt="Klipper"></p>
```

Comment:
This image doesn't show up for me locally - updating the path to "../attachments/restart.jpg" seems to resolve